GermanFilter/sections/specific.txt

```diff
@@ -104,6 +104,8 @@ focus.de##.articleContent > .textBlock > h2:empty
 wrestling-infos.de##center > a[target="_blank"]:not([rel]) > img[alt^="http"]
 golem.de##.list-articles > li[class*="li-hb-"]
 hausbau-forum.de##.samItem
+||mopo.de/assets/*/Gd/desktop/js/dmn-main.js
+mopo.de##body > div[class] > img[referrerpolicy="unsafe-url"]
```

Comment:
please change to `img[referrerpolicy="unsafe-url"]` and fix "Ad" caption under the ad

Reply:
For some reason 
`mopo.de##img[referrerpolicy="unsafe-url"]
`and
`mopo.de##div[class] > img[referrerpolicy="unsafe-url"]
`dont remove that banner on my device.

Reply:
Better check `mopo.de#@#.AD-POST`

Reply:
added